airflow/providers/amazon/aws/transfers/s3_to_redshift.py

```diff
@@ -17,8 +17,10 @@
 
 from typing import List, Optional, Union
 
+
 from airflow.models import BaseOperator
 from airflow.providers.amazon.aws.hooks.s3 import S3Hook
+from airflow.providers.amazon.aws.utils.redshift import build_credentials_block
```

Comment:
Please don't create a new file just for this -- add this method on to the Hook instead.

Reply:
We don't have a `RedshiftHook`. We are using a `PostgresHook` since redshift is based on postgres. And we hadn't really the need yet to create a hook only for redshift specific stuff. :)

Reply:
Oh yes, good point!